### DISCUSSION TITLE
Nema motor stop or/ dosesn't move by force

### DISCUSSION THREAD

**Discussion by akashgajinkar** [winder/Universal-G-Code-Sender#2006] [Help]
Sir my Nema Motor used to work with Shield v3 but now when I connect the USB to a laptop my Nema Motor stops with force and does not move it takes 11.83v with 0.188 amp and it does not move, 

I checked the resistance of the NEMA motor and its 55 ohms 

Does it mean that my motors are damaged?

But when I manually turn the shaft of my nema motor without connection I can feel shocked on my tongue, and even if I short two wires of winding I can feel the drag and resist of the shaft.

what might be the problem

For the first few days my homemade CNC worked Funtastic with Limit Switch Enabled for All 3 Axis, but things started to give Problems while setting Homing and Soft Limits, 
 Now Currently During USB is plugged In ALL 3 NEMA Motors Stop By Force taking 0.06Amp to 0.18Amp and It doesn't Move, 

**_My thinking is that My Motor has a rating of 1.2 amp and am using drv8825 so I had set Veff to 0.60  and due to this while homing it uses its full power to move all 3 axis and this might cause the issue,_** 

My code are during starting 

**** Connected to COM6 @ 115200 baud ****
Grbl 1.1h ['$' for help]
>>> $$
$0 = 0    (Step pulse time, microseconds)
$1 = 177    (Step idle delay, milliseconds)
$2 = 0    (Step pulse invert, mask)
$3 = 40    (Step direction invert, mask)
$4 = 1    (Invert step enable pin, boolean)
$5 = 0    (Invert limit pins, boolean)
$6 = 0    (Invert probe pin, boolean)
$10 = 5    (Status report options, mask)
$11 = 0.000    (Junction deviation, millimeters)
$12 = 0.000    (Arc tolerance, millimeters)
$13 = 0    (Report in inches, boolean)
$20 = 0    (Soft limits enable, boolean)
$21 = 1    (Hard limits enable, boolean)
$22 = 1    (Homing cycle enable, boolean)
$23 = 177    (Homing direction invert, mask)
$24 = 0.000    (Homing locate feed rate, mm/min)
$25 = 0.000    (Homing search seek rate, mm/min)
$26 = 0    (Homing switch debounce delay, milliseconds)
$27 = 0.000    (Homing switch pull-off distance, millimeters)
$30 = 0    (Maximum spindle speed, RPM)
$31 = 0    (Minimum spindle speed, RPM)
$32 = 1    (Laser-mode enable, boolean)
$100 = 25.000    (X-axis travel resolution, step/mm)
$101 = 105.000    (Y-axis travel resolution, step/mm)
$102 = 105.000    (Z-axis travel resolution, step/mm)
$110 = -0.000    (X-axis maximum rate, mm/min)
$111 = 0.000    (Y-axis maximum rate, mm/min)
$112 = 0.000    (Z-axis maximum rate, mm/min)
$120 = 0.000    (X-axis acceleration, mm/sec^2)
$121 = 0.000    (Y-axis acceleration, mm/sec^2)
$122 = 0.000    (Z-axis acceleration, mm/sec^2)
$130 = -0.000    (X-axis maximum travel, millimeters)
$131 = 0.000    (Y-axis maximum travel, millimeters)
$132 = 0.000    (Z-axis maximum travel, millimeters)
ok
>>> $G
[GC:G0 G54 G17 G21 G90 G94 M5 M9 T0 F0 S0]
ok
Grbl 1.1h ['$' for help]
>>> $$
[MSG:'$H'|'$X' to unlock]
$0 = 0    (Step pulse time, microseconds)
$1 = 177    (Step idle delay, milliseconds)
$2 = 0    (Step pulse invert, mask)
$3 = 40    (Step direction invert, mask)
$4 = 1    (Invert step enable pin, boolean)
$5 = 0    (Invert limit pins, boolean)
$6 = 0    (Invert probe pin, boolean)
$10 = 5    (Status report options, mask)
$11 = 0.000    (Junction deviation, millimeters)
$12 = 0.000    (Arc tolerance, millimeters)
$13 = 0    (Report in inches, boolean)
$20 = 0    (Soft limits enable, boolean)
$21 = 1    (Hard limits enable, boolean)
$22 = 1    (Homing cycle enable, boolean)
$23 = 177    (Homing direction invert, mask)
$24 = 0.000    (Homing locate feed rate, mm/min)
$25 = 0.000    (Homing search seek rate, mm/min)
$26 = 0    (Homing switch debounce delay, milliseconds)
$27 = 0.000    (Homing switch pull-off distance, millimeters)
$30 = 0    (Maximum spindle speed, RPM)
$31 = 0    (Minimum spindle speed, RPM)
$32 = 1    (Laser-mode enable, boolean)
$100 = 25.000    (X-axis travel resolution, step/mm)
$101 = 105.000    (Y-axis travel resolution, step/mm)
$102 = 105.000    (Z-axis travel resolution, step/mm)
$110 = -0.000    (X-axis maximum rate, mm/min)
$111 = 0.000    (Y-axis maximum rate, mm/min)
$112 = 0.000    (Z-axis maximum rate, mm/min)
$120 = 0.000    (X-axis acceleration, mm/sec^2)
$121 = 0.000    (Y-axis acceleration, mm/sec^2)
$122 = 0.000    (Z-axis acceleration, mm/sec^2)
$130 = -0.000    (X-axis maximum travel, millimeters)
$131 = 0.000    (Y-axis maximum travel, millimeters)
$132 = 0.000    (Z-axis maximum travel, millimeters)
ok
>>> $G
[GC:G0 G54 G17 G21 G90 G94 M5 M9 T0 F0 S0]
ok
>>> $X
[MSG:Caution: Unlocked]
ok
>>> $J=G21G91X10F200
>>> $J=G21G91X10F200
>>> $J=G21G91X-10F200

Well While Loading any file and while playing it it says "Error While Starting While File Stream"

**Comment by ElsGear** [comment]
Hi, just a simple test to check if it is a nema problem.
Remove the stepper from the system and test it on something else with step/dir signals.
Or change the stepper for a spare one.

If it's not the problem you can focus on the software and controller.
